api/v1beta1/helmchart_types.go

```diff
@@ -102,6 +110,9 @@ type LocalHelmChartSourceReference struct {
 	Kind string `json:"kind"`
 
 	// Name of the referent.
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=63
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxLength=253
```

Kubernetes object name max length is 253, please use this value for all fields that references a Kube object by name.